flink-connector-kafka/src/test/resources/log4j2-test.properties

```diff
@@ -27,15 +27,41 @@ appender.testlogger.target = SYSTEM_ERR
 appender.testlogger.layout.type = PatternLayout
 appender.testlogger.layout.pattern = %-4r [%t] %-5p %c %x - %m%n
 
-logger.kafka.name = kafka
+# Overwrite the level for all Flink related loggers
+logger.flink.name = org.apache.flink
+logger.flink.level = OFF # WARN for starting debugging
+logger.flinkconnector.name = org.apache.flink.connector
+logger.flinkconnector.level = OFF # INFO/DEBUG for starting debugging
+
+# Kafka producer and consumer level
+logger.kafka.name = org.apache.kafka
 logger.kafka.level = OFF
-logger.kafka2.name = state.change
+
+# For old school kafka test that spawn the test server in the same JVM
+logger.kafka2.name = kafka
```

Comment:
Which tests still spawn kafka in the same JVM?

Reply:
Errr it seems none. Good catch.

We still have test dependencies on kafka server and zookeeper that I can probably remove and with it these log confs.